.devcontainer/devcontainer.json

```diff
@@ -1,20 +1,7 @@
-// See https://github.com/julia-vscode/julia-devcontainer/blob/master/Dockerfile for image contents
 {
-	"name": "Julia (Community)",
-	"image": "ghcr.io/julia-vscode/julia-devcontainer",
-
-	// Configure tool-specific properties.
-	"customizations": {
-		// Configure properties specific to VS Code.
-		"vscode": {
-			// Add the IDs of extensions you want installed when the container is created.
-			"extensions": [
-				"julialang.language-julia"
-			]
-		}
+	"image": "mcr.microsoft.com/vscode/devcontainers/base",
+	"features": {
+		"ghcr.io/julialang/devcontainer-features/julia:1": {},
```

Comment:
Does this come with the Julia extension?

Reply:
Yes, it's built in